fedmsg_meta_fedora_infrastructure/autocloud.py

```diff
@@ -85,7 +86,10 @@ def handle_v2_autocloud_subtitle(self, msg, **config):
                                 compose_id=compose_id)
 
         if "autocloud.compose" in msg['topic']:
-            compose_id = msg['msg']['id']
+            compose_id = msg['msg'].get('id', '')
+
+            if compose_id is '' and "complete" in msg['topic']:
+                compose_id = msg['msg'].get('compose_id', '')
 
```

Comment:
+1 for the default

But do we need that check for 'complete', we could just do something like: 
``compose_id = msg['msg'].get('id', compose_id = msg['msg'].get('compose_id', 'A compose'))``
thoughts?